collection/nilupul.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "nilupul",
+    "meaning": "Sinhala for \"Blue Lotus\" ",
+    "translations": {
+        "si": "නිලුපුල්",
```

Comment:
Can you please update the translations to use the correct ISO 639-3 code instead two letter codes?
List of ISO codes can be found [here](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).